website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -55,6 +55,9 @@ Currently, this endpoint returns a full, unpaginated list of organizations (with
 
 | Parameter      | Description                                                                   |
 | -------------- | ----------------------------------------------------------------------------- |
+| `q`            | **Optional.** A search query string. Organizations are searchable by name and notification email. This takes precedence over the attribute specific searches `q[email]` or `q[name]`. |
+| `q[email]`     | **Optional.** A search query string. This will search organizations by notification email. If used with `q[name]`, it will return organizations that match both queries. |
+| `q[name]`      | **Optional.** A search query string. This will search organizations by name. If used with `q[email]`, it will return organizations that match both queries. |
```

Comment:
```suggestion
| `q`            | **Optional.** A search query string. Organizations are searchable by name and notification email. This query takes precedence over the attribute specific searches `q[email]` or `q[name]`. |
| `q[email]`     | **Optional.** A search query string. This query searches organizations by notification email. If used with `q[name]`, it returns organizations that match both queries. |
| `q[name]`      | **Optional.** A search query string. This query searches organizations by name. If used with `q[email]`, it returns organizations that match both queries. |
```

Suggest adding "query" after each "This" to avoid a "This" without a subject. Also suggest changing "will search" to searches" and "will return" to "returns" to avoid future tense.

Reply:
@mgarrell777 Thanks for the feedback. Since this language was taken directly from the [enterprise/api-docs](https://www.terraform.io/enterprise/api-docs/admin/organizations) for admin/orgs, wondering if you suggest I also update the language there too? Keep everything uniform.

Reply:
Looks like that language lives in the [ptfe-releases](https://github.com/hashicorp/ptfe-releases/blob/main/website/docs/enterprise/api-docs/admin/organizations.mdx) repo.

Reply:
Hey @laurenolivia - good catch! Just to make _sure_ I get what's happening - you're adding some functionality to the TFC API (these query parameters) that previously only the TFE admin API had. And you pulled the language directly from the TFE Admin API. Yes?

If so then yes, please also update this language on that page in the `ptfe-releases` repository. You are correct that the TFE Admin API lives there now. Please tag @mgarrell777 on that PR when it's available. Thank you - we appreciate it!

Reply:
@laurapacilio Correct! Currently, the admin/org endpoint in TFE permits querying on certain parameters that we are now extending to the /organizations endpoint TFC via this PR #[13125](https://github.com/hashicorp/atlas/pull/13125). Since I pulled the language from admin/org in TFE, I will update the language in both locations.

And I will tag @mgarrell777!